evalml/tests/component_tests/decomposer_tests/test_decomposer.py

```diff
@@ -444,27 +444,16 @@ def test_decomposer_set_period(decomposer_child_class, period, generate_seasonal
     "decomposer_child_class",
     decomposer_list,
 )
-@pytest.mark.parametrize(
-    "periodicity_determination_method",
-    [
-        "autocorrelation",
-        pytest.param(
-            "partial-autocorrelation",
-            marks=pytest.mark.xfail(reason="Partial Autocorrelation not working yet."),
-        ),
-    ],
-)
 @pytest.mark.parametrize("decomposer_picked_correct_degree", [True, False])
 @pytest.mark.parametrize(
     "synthetic_data,trend_degree,period",
-    [*itertools.product(["synthetic"], [1, 2, 3], [7, 30, 365]), ("real", 1, 365)],
+    [*itertools.product(["synthetic"], [1, 2, 3], [1, 7, 30, 365]), ("real", 1, 365)],
 )
 def test_decomposer_determine_periodicity(
```

Comment:
I hate to be a jerk, particularly because this is initially my failure to add - but can we include either enhancing this test or add a separate one calling out the no-seasonality results?

Reply:
Ah, here I added the `1` to the period parametrization to represent that, since a seasonal period of 1 is identical to having no seasonal period. I'll switch this over to being explicitly `None` though.